hack/test/unit

```diff
@@ -41,12 +41,25 @@ if [ -n "${libnetwork_pkg_list}" ]; then
 	# libnetwork tests invoke iptables, and cannot be run in parallel. Execute
 	# tests within /libnetwork with '-p=1' to run them sequentially. See
 	# https://github.com/moby/moby/issues/42458#issuecomment-873216754 for details.
-	gotestsum --format=standard-quiet --jsonfile=bundles/go-test-report-libnetwork.json --junitfile=bundles/junit-report-libnetwork.xml -- \
-		"${BUILDFLAGS[@]}" \
-		-cover \
-		-coverprofile=bundles/coverage-libnetwork.out \
-		-covermode=atomic \
-		-p=1 \
-		${TESTFLAGS} \
-		${libnetwork_pkg_list}
+
+	if [ -n "${DELVE_PORT:-}" ]; then
+		dlv --listen="0.0.0.0:${DELVE_PORT##*:}" \
```

Comment:
Looking at this; when running through Delve, we skip gotestsum. Would it be possible somehow to run "both", or no way to wrap it twice?

I also notice that `BUILDFLAGS` is omitted in this branch; is that problematic?

Reply:
> Looking at this; when running through Delve, we skip gotestsum. Would it be possible somehow to run "both", or no way to wrap it twice?

Honestly, I wrote this patch almost a year ago and I check it out from time to time. From what I recall, there's no way to debug a test run through gotestsum + delve. If you try to do this, you're going to debug `gotestsum` itself, and not the actual test. I think it wouldn't provide much value anyway as I don't expect it to be used without targetting a specific test.

> I also notice that BUILDFLAGS is omitted in this branch; is that problematic?

Oh, right! I guess I was too lazy to find out the right way to escape strings to put them in `--build-flags=...`. I'll fix that.

Reply:
I don't know what syntax/escaping it expects (I am still not a Delve user, sorry!), but since we're in Bash here you might be able to use/abuse `%q` in a `printf` something like `buildFlags="$(printf '%q\n' "${BUILDFLAGS[@]}")" ... --build-flags="$buildFlags"`? :eyes:

```console
$ printf '%q\n' '--foo=foo foo' '--bar=bar bar' '--baz=baz baz'
--foo=foo\ foo
--bar=bar\ bar
--baz=baz\ baz
```